src/main/java/org/opensearch/flowframework/workflow/WorkflowStepFactory.java

```diff
@@ -118,6 +118,10 @@ public WorkflowStepFactory(
         stepMap.put(CreateIngestPipelineStep.NAME, () -> new CreateIngestPipelineStep(client, flowFrameworkIndicesHandler));
         stepMap.put(DeleteIngestPipelineStep.NAME, () -> new DeleteIngestPipelineStep(client));
         stepMap.put(CreateSearchPipelineStep.NAME, () -> new CreateSearchPipelineStep(client, flowFrameworkIndicesHandler));
+        stepMap.put(UpdateIngestPipelineStep.NAME, () -> new UpdateIngestPipelineStep(client));
+        stepMap.put(UpdateSearchPipelineStep.NAME, () -> new UpdateSearchPipelineStep(client));
+        stepMap.put(UpdateIndexStep.NAME, () -> new UpdateIndexStep(client));
+
```

Comment:
does the new GetResourceStep (or whatever it may be renamed to) need to be here?

Reply:
This can be removed